server/src/resources/employee/employee.router.js

```diff
@@ -3,9 +3,11 @@ import { controllers } from './employee.controllers';
 
 const router = Router();
 
+
+
 router
   .route('/')
-  .get(controllers.getMany)
+  .get(controllers.getEmployees)
```

Comment:
you still want this to be named `getMany`. Just overwrite the the class in the employees controller.